### ISSUE TITLE
Upgrade to RC2 of dotnet core

### ISSUE THREAD

**Issue by bilal-fazlani** [aws/aws-sdk-net#356]

**Comment by normj** [comment]
Working on it
https://github.com/aws/aws-sdk-net/commits/netcore-development

**Comment by bilal-fazlani** [comment]
Awesome!

**Comment by bilal-fazlani** [comment]
@normj 

After I upgraded my projects to RC2 everything seemed to work fine.. except for one thing.

Some of my unit tests started throwing failing and throwing this exception:

```
[13:59:54][Step 1/3]       System.IO.FileNotFoundException : Could not load file or assembly 'DynamoDBv2, Version=3.2.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.
[13:59:54][Step 1/3]       
[13:59:54][Step 1/3]       Stack Trace:
[13:59:54][Step 1/3]            at System.ModuleHandle.ResolveType(RuntimeModule module, Int32 typeToken, IntPtr* typeInstArgs, Int32 typeInstCount, IntPtr* methodInstArgs, Int32 methodInstCount, ObjectHandleOnStack type)
[13:59:54][Step 1/3]            at System.ModuleHandle.ResolveTypeHandleInternal(RuntimeModule module, Int32 typeToken, RuntimeTypeHandle[] typeInstantiationContext, RuntimeTypeHandle[] methodInstantiationContext)
[13:59:54][Step 1/3]            at System.Reflection.RuntimeModule.ResolveType(Int32 metadataToken, Type[] genericTypeArguments, Type[] genericMethodArguments)
[13:59:54][Step 1/3]            at System.Reflection.CustomAttribute.FilterCustomAttributeRecord(CustomAttributeRecord caRecord, MetadataImport scope, Assembly& lastAptcaOkAssembly, RuntimeModule decoratedModule, MetadataToken decoratedToken, RuntimeType attributeFilterType, Boolean mustBeInheritable, Object[] attributes, IList derivedAttributes, RuntimeType& attributeType, IRuntimeMethodInfo& ctor, Boolean& ctorHasParameters, Boolean& isVarArg)
[13:59:54][Step 1/3]            at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeModule decoratedModule, Int32 decoratedMetadataToken, Int32 pcaCount, RuntimeType attributeFilterType, Boolean mustBeInheritable, IList derivedAttributes, Boolean isDecoratedTargetSecurityTransparent)
[13:59:54][Step 1/3]            at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeFieldInfo field, RuntimeType caType)
[13:59:54][Step 1/3]            at System.Attribute.GetCustomAttributes(MemberInfo element, Boolean inherit)
[13:59:54][Step 1/3]            at FluentValidation.ValidatorOptions.GetDisplayName(MemberInfo member)
[13:59:54][Step 1/3]            at FluentValidation.Internal.PropertyRule.GetDisplayName()
[13:59:54][Step 1/3]            at FluentValidation.Internal.PropertyRule.<Validate>d__60.MoveNext()
[13:59:54][Step 1/3]            at System.Linq.Enumerable.<SelectManyIterator>d__157`2.MoveNext()
[13:59:54][Step 1/3]            at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
[13:59:54][Step 1/3]            at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
[13:59:54][Step 1/3]            at FluentValidation.AbstractValidator`1.Validate(ValidationContext`1 context)
[13:59:54][Step 1/3]            at Notifications.Messaging.Recommendations.Tests.Models.JobDetailsValidatorTest.AdvertiserNameShouldNotBeUndefined()
```

This happened only after upgrading to RC2 and only for `DynamoDBv2` all the other packages are fine. 


And my project for json clearly states `"AWSSDK.DynamoDBv2": "3.2.3.1-beta"`
Not sure why it started looking for version 3.2.0.0 after upgrading to RC2. 

Its the only thing right now stopping me from going live with RC2. Hopefully after you upgrade this library.., this should be fixed ?

**Comment by normj** [comment]
Yeah I think I noticed that in during the upgrade. The method System.Reflection.CustomAttribute.GetCustomAttributes which the SDK calls was moved to a different package in RC2.

**Comment by mcraiha** [comment]
I know this is a bit off topic, but could I get some build instructions for dotnet core?

**Comment by bilal-fazlani** [comment]
Any ETA on this one ?

**Comment by normj** [comment]
I just pushed version 3.2.4-beta to NuGet which is updated for RC2.

Sorry I got delayed a couple days. The master branch had a new credential and region picker feature release that I wanted to get included in the beta branch since it will be useful for setting credentials on non-windows environments. Here is the blog post on that feature.

https://blogs.aws.amazon.com/net/post/Tx2HQ4JRYLO7OC4/Updates-to-Credential-and-Region-Handling
